recipes/notolog/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "notolog" %}
```

Comment:
This has to be published as a pre-release build: https://conda-forge.org/docs/maintainer/knowledge_base/#pre-release-builds

Reply:
Released to stable build atm.